README.adoc

```diff
@@ -1,32 +1,19 @@
-= Address Book (Level 4)
+= Hotel Management System
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
 https://travis-ci.org/se-edu/addressbook-level4[image:https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master[Build Status]]
 https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
 https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
 https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&utm_medium=referral&utm_content=se-edu/addressbook-level4&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a[Codacy Badge]]
 
-ifdef::env-github[]
-image::docs/images/Ui.png[width="600"]
-endif::[]
-
-ifndef::env-github[]
-image::images/Ui.png[width="600"]
```

Comment:
The picture doesn't have to be a screenshot of the actual product. A mockup is totally fine as well so don;t forget to add it in.